### PR TITLE
Person page FE

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1379069797)

### Description of Changes Made

- Styling for the person page
- Fixes focus styles for listings
- Adds a new listing component - listing--simple - for the blog posts on the team page

### How to Test

View any team page, e.g. http://localhost:8000/team/abigail-hampson/ (first you will need to add some items to the team taxonomy, and then edit abigail's page to set a team in the promote tab

In the pattern library:
http://localhost:8000/pattern-library/pattern/patterns/molecules/listing/listing--simple.html - new listing
http://localhost:8000/pattern-library/pattern/patterns/pages/team/team_detail.html - team page

### Screenshots

<details>
  <summary>Expand to see more</summary>

New blog listing style

Screenshot 2024-02-14 at 12 21 06

Top area of page
Screenshot 2024-02-14 at 12 21 30
</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [x] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [x] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [ ] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [x] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/pages/team/team_detail.html [hunk 1]
```diff
@@ -2,27 +2,54 @@
...
+                    {% endfor %}
+                </div>
+                <span class="team-page-meta__role">{{ page.role }}</span>
+            </div>
+        </div>
 
-    {{ page.intro|richtext }}
-    {{ page.biography|richtext }}
+        <div class="image grid__team-image">
+            {% srcset_image page.image format-webp width-{400,650,950} sizes="(max-width: 598px) 400px, (max-width: 1022px) 950px, (min-width: 1023px) 650px" %}
```
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/pages/team/team_detail.html:20]: I was chatting to Andy the other day, and apparently the new wagtail image tags are way more efficient than creating the renditions individually, so I have experimented here and may try and update other images in the build if we have time. I've managed to get it working in the pattern library yaml, too.

File: tbx/static_src/sass/components/_listing.scss [hunk 2]
```diff
@@ -5,17 +5,21 @@
 .listing {
     list-style-type: none;
 
+    &__link {
```
Comment (helenb) [inline tbx/static_src/sass/components/_listing.scss:8]: The focus styles weren't working on listings and this fixes that.

File: tbx/project_styleguide/templates/patterns/molecules/listing/listing--simple.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<li class="listing-simple listing">
+    <a class="listing__link" href="{{ link }}" aria-label="{{ aria_text|default:title }} by {{ name }}">
+        <h2 class="listing__title">
+            {# The title and icon need to be on the same line with no whitespace to prevent the arrow being orphaned on a new line #}
+            <span class="listing__title-text">{{ title }}</span><span class="listing__tail">&nbsp;{% include "patterns/atoms/icons/icon.html" with name="arrow-wide" classname="listing-avatar__arrow listing__arrow" %}</span>
+        </h2>
+    </a>
+    <div class="listing__details">
+        <div>
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/listing/listing--simple.html:9]: Might be able to remove this `<div>` if it's not doing anything?

File: tbx/project_styleguide/templates/patterns/pages/team/team_detail.html (hunk 1 above)
Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/pages/team/team_detail.html:20]: Ahh that's awesome. Well done on getting it working in the Pattern Library as well!

**Review by chris-lawton — APPROVED** [review]
Nice work @helenb - looks great!

File: tbx/project_styleguide/templates/patterns/molecules/listing/listing--simple.html (hunk 3 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/listing/listing--simple.html]: sorted.

**Pull request merged**